thermochem/janaf.py

```diff
@@ -41,42 +43,51 @@ class JanafPhase(object):
     It reads in the JANAF data file and produces functions which interpolate
     the thermodynamic constants.
 
+    Tr stands for reference temperature and is 298.15 K
+
     >>> db = Janafdb()
     >>> p = db.getphasedata(formula='O2Ti', name='Rutile', phase='cr')
     >>> p.cp([500, 550, 1800]).astype(int).tolist()
     [67, 68, 78]
-    >>> print(p.S([500, 550, 1800]))
+    >>> print(p.S([500, 550, 1800]))        # Entropy in J/mol/K
     [  82.201    88.4565  176.876 ]
-    >>> print(p.gef([500, 550, 1800]))
+    >>> print(p.gef([500, 550, 1800]))      # [G-H(Tr)]/T in J/mol/K
     [  57.077   59.704  115.753]
-    >>> print(p.hef([500, 550, 1800]))
+    >>> print(p.hef([500, 550, 1800]))      # H-H(Tr) in kJ/mol
     [  12.562    15.9955  110.022 ]
-    >>> print(p.DeltaH([500, 550, 1800]))
+    >>> print(p.DeltaH([500, 550, 1800]))   # Gibbs free energy in kJ/mol
     [-943670.  -943229.5 -936679. ]
-    >>> print(p.DeltaG([500, 550, 1800]))
+    >>> print(p.DeltaG([500, 550, 1800]))   # Helmholtz free enegy in kJ/mol
     [-852157.  -843046.5 -621013. ]
-    >>> p.logKf([500, 550, 1800]).astype(int).tolist()
+    >>> p.logKf([500, 550, 1800]).astype(int).tolist() # Equilibrium constant of formation.
     [89, 80, 18]
-    >>> print(p.cp(50000))
+    >>> print(p.cp(1000))                   # Heat capacity in J/mol/K
+    [74.852]
+    >>> print(p.cp(50000))                  # Example of erroneous extrapolation.
     Traceback (most recent call last):
         ...
     ValueError: A value in x_new is above the interpolation range.
     """
 
+    # TODO On these docstrings, they actually print array([89,90,18]) rather than [89,90,18].  Is this going to throw off CI?
+
```

Comment:
The docstrings for the array worked fine, but the ones with exceptions were giving me trouble

Reply:
OK.  I made a new commit with improved docstrings.